airflow/README.md

```diff
@@ -34,34 +34,23 @@ gcloud init
 # gcloud auth application-default login
 ```
 
-To run the gtfs_downloader dags, it is required to generate a yml file of the agencies with various
-secrets filled in. To run this, do the following:
-
-1. Setup a virtual environment within the `script` folder by running `python -m venv .venv` in the `script` folder.
-2. Install the needed requirements via `pip install -r requirements.txt` (the requirements in `script`)
-3. Copy `airflow/data/example-secrets.csv` to `airflow/data/secrets.csv` and fill in the secret keys as needed
-4. run `python yml_convert.py ../airflow/data/agencies.yml ../airflow/data/agencies.filled.yml ../airflow/data/secrets.csv`
-5. Copy `/airflow/data/agencies.yml` to `/airflow/data/agencies_raw.yml`
-
-Next, run the initial database migration and create an `airflow / airflow` user to debug with:
-
-```console
+Next, run the initial database migration which also creates a default user named `airflow.
+```shell
 docker-compose run airflow db init
 ```
 
-Next, start all services with:
-
+Next, start all services including the Airflow web server.
 ```console
 docker-compose up
 ```
 
 To access the web UI, visit `http://localhost:8080`.
 The default login and password for airflow's image are both "airflow".
 
-To run a DAG, you can either test it via the web UI or run a one-off with:
+You may execute DAGs via the web UI, or just specific individual tasks via the CLI.
 
 ```console
-docker-compose run airflow dags trigger <dag_id>
+docker-compose run airflow tasks test download_gtfs_schedule_v2 download_schedule_feeds 2022-04-01T00:00:00
```

Comment:
Do people need access to Secrets Manager to run the downloader DAG? How does that work in local Airflow?